### ISSUE TITLE
gitbuilder should be able to build multiple projects

### ISSUE THREAD

**Issue by n0ha** [apenwarr/gitbuilder#1]
Right now, when I want to build multiple projects, I need to have multiple gitbuilder instances, because there is only one build dir.

**Comment by apenwarr** [comment]
True. The correct fix for this is to actually have multiple gitbuilders, but then be able to merge the results into a single viewer, I think, so I'll leave this open in the hope that we actually produce this someday.
